mkosi.images/system/mkosi.conf.d/10-opensuse/mkosi.build.chroot

```diff
@@ -40,9 +40,7 @@ DIST="$(rpm --eval %dist)"
 ARCH="$(rpm --eval %_arch)"
 SRCDEST="/usr/src/debug/systemd-$VERSION-${RELEASE}${DIST}.$ARCH"
 
-# TODO: Enable this when the opensuse spec stops unconditionally setting FORTIFY_SOURCE=2.
-# EXTRA_CFLAGS="-O${OPTIMIZATION:-0} -Wp,-U_FORTIFY_SOURCE"
-EXTRA_CFLAGS=""
+EXTRA_CFLAGS="-O${OPTIMIZATION:-0} -Wp,-U_FORTIFY_SOURCE"
```

Comment:
I wonder why `_FORTIFY_SOURCE` is always dropped? `_FORTIFY_SOURCE=3` is great in that it can detect things ASan/UBSan can't find (or more precisely they tend to fail down the line and it's hard to track down where it all goes sideways initially). I think in terms of testing it would be nice to build systemd with `_FORTIFY_SOURCE=2` and `_FORTIFY_SOURCE=3` on top of ASan/UBSan jobs.

Reply:
Can you please open issues for these? I'm happy to fix them but it's much easier to keep track if you open issues rather then commenting on PRs.

Reply:
I comment on PRs because when I look at commits I go to GitHub first to see how they were reviewed and why they were merged :-) I agree that it would be easier to keep track of all that if I opened issues. Sorry!